src/DotNetty.Handlers/Tls/TlsHandler.cs

```diff
@@ -710,8 +710,8 @@ public void ExpandSource(int count)
 
                 ArraySegment<byte> sslBuffer = this.sslOwnedBuffer;
 
-#if NETSTANDARD1_3
-                this.readByteCount = this.ReadFromInput(sslBuffer.Array, sslBuffer.Offset, sslBuffer.Count);
+#if NETSTANDARD1_3 || NETSTANDARD2_0
+				this.readByteCount = this.ReadFromInput(sslBuffer.Array, sslBuffer.Offset, sslBuffer.Count);
                 // hack: this tricks SslStream's continuation to run synchronously instead of dispatching to TP. Remove once Begin/EndRead are available. 
```

Comment:
Begin/End Read/Write in already added in NETSTANDARD2_0, so maybe override them in this class(and the one in test)?

Reply:
@yyjdelete Exactly! There are things need to be cleaned up specifically for netstandard 2.0 before we officially support it. Supporting both 1.3 and 2.0 would make some part of code much harder to maintain. Plus, there are so many things we can take advantage of in netstandard 2.0

Reply:
I can attempt to help with that transition, but I'm not all that familiar with the internals of DotNetty or what specifically netstandard 2.0 added (other than a lot).
To me it seems like a larger project that should be broken out into pieces (perhaps starting with multi-target as the community works on the rest). It also seems that DotNetty is due for a release before that project gets started (would be really nice to get that http pr merged).